docker/configure.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+# This script is responsible for generating random passwords for your database
+# and setting up your environment files
+# You should only ever run this script once.
+
+if ! [ -n "$BASH_VERSION" ];then
+    echo "this is not bash, calling self with bash....";
+    SCRIPT=$(readlink -f "$0")
+    /bin/bash $SCRIPT
+    exit;
+fi
+
+if [ -d $HOME/polr-data ]; then
+    echo "Unfortunately, it appears you already have polr data in $HOME/polr-data."
+    echo "This script sets new passwords, which would make the database inaccessible."
+    echo "If you wish to proceed, please delete that directory and re-run this script."
+    exit 1
+fi
+
+
+# Execute everything from a relative point of where
+# this script is stored.
+DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+cd $DIR
+
+
+
+
+# Generate password and replace if found
+# Repeating this once already done does nothing
+# which is a good thing.
+ROOT_PASSWORD=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
+POLR_USER_PASSWORD=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
+
+# If there is no .env file, then we assume we have not copied
+# .env.setup to .env yet.
+echo "creating the polr .env file"
+cp $DIR/../.env.setup $DIR/.env
+
+FILEPATH="$DIR/.env"
```

Comment:
Did you consider @fagiani's comment of just setting the environment vars instead of editing the env file?

Reply:
I had to re-read his comment a few times but I think I got it now. I am happy to remove this line now:

```
cat /var/www/polr/.env.setup > /data/env
```

I don't need it. What I'm doing now is using that template file to create the .env file inside the docker container instead which sets environment variables, rather than me overwriting your .env file which is one level higher (outside the docker container). 

I am using a separate .env file (maybe I should rename it) that the docker-compose.yml file links to for both the application container and the mysql database. The user is free to update the docker-compose.yml file to remove those references and put in their own env lines but I guess the downside of this is that it will show up as a chnage and cause issues when performing updats with docker pull? Perhaps the `docker-compose.yml` file needs to be `docker-compose.yml.tmpl` and the user needs to rename it to `docker-compose.yml` with their desired changes and it be added to the `.gitignore` file? I like the idea of the bash script(s) doing this for the user if they just want a fast deploy option, but they can manually do things without the script(s) if they want to just use the docker-compose up command?